config-qemu-aarch64

```diff
@@ -0,0 +1,343 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Unikraft/0.13.1~e467cdf6 Configuration
+#
+CONFIG_UK_FULLVERSION="0.13.1~e467cdf6"
+CONFIG_UK_CODENAME="Atlas"
+CONFIG_UK_ARCH="arm64"
+CONFIG_UK_BASE="/home/joachim/Desktop/unikraft/ELF/duktape/.unikraft/unikraft"
+CONFIG_UK_APP="/home/joachim/Desktop/unikraft/ELF/duktape"
+CONFIG_UK_DEFNAME="duktape"
+
+#
```

Comment:
Remove commented out configuration options. Only keep the non-commented lines for configuration options (i.e. lines starting with `CONFIG_...`).